src/component/routers/GroupsRouter.tsx

```diff
@@ -87,12 +88,12 @@ export class GroupsRouter extends React.PureComponent<GroupsRouterProps> {
 }
 
 const mapStateToProps = ({
-    flowContext: { groups },
+    flowContext: { groups: localGroups },
     flowEditor: { editorUI: { translating } },
     nodeEditor: { nodeToEdit }
 }: AppState) => ({
     translating,
-    groups,
+    localGroups,
```

Comment:
changing this from `groups` to `localGroups` to be more specific about the purpose of the prop. perhaps we should do the same in the store for locals?